doc/cephadm/operations.rst

```diff
@@ -524,18 +524,18 @@ Purging a cluster
 
 .. danger:: THIS OPERATION WILL DESTROY ALL DATA STORED IN THIS CLUSTER
 
-In order to destroy a cluster and delete all data stored in this cluster, pause 
-cephadm to avoid deploying new daemons.
+In order to destroy a cluster and delete all data stored in this cluster, disable
+cephadm to stop all orchestration operations (so we avoid deploying new daemons).
 
-.. prompt:: bash #
+.. prompt:: bash (cephadm shell session)#
```

Comment:
in the readthedocs render this (and the other prompt) aren't showing up

![image](https://user-images.githubusercontent.com/47704447/151552721-9b709ebe-e29d-402d-afa4-2bfc6f17e05a.png)

possibly caused by putting "(cephadm shell session)" here

Reply:
https://ceph--44810.org.readthedocs.build/en/44810/cephadm/operations/#purging-a-cluster

Reply:
Fixed now 
![image](https://user-images.githubusercontent.com/47704447/151976256-c01e0eaa-dd83-43f5-b45c-1631283758f5.png)